netdata-installer.sh

```diff
@@ -26,10 +26,26 @@ if [ "${NETDATA_SOURCE_DIR}" != "${INSTALLER_DIR}" ] && [ "${INSTALLER_DIR}" !=
 fi
 
 # -----------------------------------------------------------------------------
-# Pull in OpenSSL properly if on macOS
-if [ "$(uname -s)" = 'Darwin' ] && [ -d /usr/local/opt/openssl/include ]; then
-  export C_INCLUDE_PATH="/usr/local/opt/openssl/include"
-  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
+# Pull in OpenSSL and other includes and libs properly if on macOS
+if [ "$(uname -s)" = 'Darwin' ]; then
```

Comment:
@MrZammler FWIW, I think it would be better to use `HOMEBREW_PREFIX` to resolve these paths. If the variable is missing we could try to get it by using `brew`'s `shellenv`. Having said that I think the PR looks good.